README.md

```diff
@@ -24,7 +24,15 @@ PowerGSLB is a simple DNS based Global Server Load Balancing (GSLB) solution.
 * [Building PowerGSLB RPM packages](#building-powergslb-rpm-packages)
 * [Using PowerGSLB Docker image](#using-powergslb-docker-image)
 * [Building PowerGSLB Docker image](#building-powergslb-docker-image)
-
+* [Install using VirtualEnv](#install-using-virtualenv)
+* [Nginx as reverse proxy](#nginx-as-reverse-proxy)
```

Comment:
I think it will not get worse if there will be documentation about nginx here? nginx is very flexible, fast and secure. As for me I prefer to use it for protecting PowerGSLB from the world.